src/PrestaShopBundle/Form/Admin/Product/ProductCombinationBulk.php

```diff
@@ -94,6 +94,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('minimal_quantity', 'Symfony\Component\Form\Extension\Core\Type\NumberType', array(
                 'required' => false,
                 'label' => $this->translator->trans('Minimum quantity', [], 'Admin.Catalog.Feature'),
+            ))
+            ->add('images', 'Symfony\Component\Form\Extension\Core\Type\ChoiceType', array(
+                'required' => false,
+                'multiple' => true,
+                'label' => $this->translator->trans('Image', [], 'Admin.Catalog.Feature'),
```

Comment:
Can you please use "Admin.Global" as a domain? Thanks!

Reply:
Done!